store/tikv/region_request.go

```diff
@@ -167,6 +168,14 @@ func (s *RegionRequestSender) sendReqToRegion(bo *Backoffer, ctx *RPCContext, re
 	if e := tikvrpc.SetContext(req, ctx.Meta, ctx.Peer); e != nil {
 		return nil, false, errors.Trace(e)
 	}
+	// judge the store limit switch.
+	if atomic.LoadUint32(&config.GetGlobalConfig().StoreLimit) > 0 {
+		if err := s.getStoreToken(ctx.Store); err != nil {
+			logutil.BgLogger().Warn("get store token failed", zap.Error(err))
+			return nil, false, errors.Trace(err)
```

Comment:
The error returned from `getStoreToken` has contained trace info, so we can return error directly instead of `errors.Trace` again.

Reply:
done.